lib/registries/addon/components/link/component.ts

```diff
@@ -0,0 +1,67 @@
+import { layout, tagName } from '@ember-decorators/component';
+import { action, computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import { isArray } from '@ember/array';
+import Component from '@ember/component';
+import { assert } from '@ember/debug';
+import RouterService from '@ember/routing/router-service';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import template from './template';
+
+type AnchorRel = 'noreferrer' | 'noopener';
+type AnchorTarget = '_self' | '_blank' | '_parent' | '_top';
+
+@tagName('')
+@layout(template)
+export default class Link extends Component {
```

Comment:
This is my favorite custom link component so far (I think we're up to 4 or 5?). Maybe we can promote it and delete the others...